les_2/HW_2/task_hw_2_1.py

```diff
@@ -0,0 +1,25 @@
+# Напишите программу, которая получает целое
+# число и возвращает его шестнадцатеричное
+# строковое представление. Функцию hex
+# используйте для проверки своего результата.
+
+BASE = 16
+
+
+def convert(input_number: int) -> str:
+    output_number = []
+    translate_to_hex = {10:'a', 11:'b', 12:'c', 13:'d', 14:'e', 15:'f'}
```

Comment:
Можно добавить остальные ключи-цифры из этой системы или вообще использовать строку
```suggestion
    translate_to_hex = '0123456789abcdef'
```